### ISSUE TITLE
[backend-js] Build an additional bundle importable via <script> tag

### ISSUE THREAD

**Issue by dstpierre** [staticbackendhq/core#81]
The [JavaScript client library](https://github.com/staticbackendhq/backend-js) currently only build a bundle that can be `imported`, for instance in a React application via:

```js
import { Backend } from "@staticbackend/js";
```
It could be nice to have an option to directly use the library via a `<script>` tag in vanilla JavaScript, a bit like Alpine.js or htmx:

```html
<script src="/some/path/backend.js"></script>
```
I'm not certain if it's possible, I think it's what is called a IIFE. Do we need to update the build system.

In short, I could use the help of a more experienced front-end developer with everything bundling related.

**Comment by dstpierre** [comment]
I've added the `iife` output for the v1.5.0-alpha.0
